nncf/experimental/common/graph/netron.py

```diff
@@ -31,17 +32,29 @@ class Tags:
 
 
 class PortDesc:
-    def __init__(self, port_id: str, shape: Optional[List[int]] = None, precision: str = None):
+    """
```

Comment:
Could you please reformat docstrings according to [this](https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md#422-functions-and-methods) example?
More examples can be found [here].(https://github.com/openvinotoolkit/nncf/blob/develop/nncf/quantization/fake_quantize.py)